dist/Dockerfile

```diff
@@ -1,10 +1,7 @@
 FROM centos:7
 
-RUN yum -y groupinstall 'Development Tools'
-RUN yum -y install openssl-devel
```

Comment:
have you tested resulting image with this?

Reply:
I tested the steps from the docker-based stack outlined in #20 and it works as expected

Reply:
Ideally our CI would test that ;-)